lib/experimental/Widgets/Layout/HomeLayout/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Children, forwardRef, ReactNode } from "react"
+import { useMediaQuery } from "usehooks-ts"
+import { WidgetStrip } from "../WidgetStrip"
+
+type Props = {
+  widgets?: ReactNode[]
+  children?: ReactNode
+}
+
+export const HomeLayout = forwardRef<HTMLDivElement, Props>(function Dashboard(
+  { widgets, children },
+  ref
+) {
+  const isSmallerScreen = useMediaQuery("(max-width: 992px)", {
+    initializeWithValue: true,
+  })
```

Comment:
What if instead of checking the media query we check for the width of the element? That's very easy to achieve with `useResizeObserver` which comes bundled with the `usehooks-ts` library we already have:

https://usehooks-ts.com/react-hook/use-resize-observer

Reply:
Done! ✅